src/me.rs

```diff
@@ -92,14 +92,13 @@ mod nasm {
   }
 
   #[target_feature(enable = "ssse3")]
-  unsafe fn sad_ssse3<T: Pixel>(
-    plane_org: &PlaneSlice<'_, T>, plane_ref: &PlaneSlice<'_, T>, blk_h: usize,
+  unsafe fn sad_ssse3(
```

Comment:
Perhaps we can rename `sad_ssse3` to `sad_hbd_ssse3` for clarity.

Reply:
hbd?

Reply:
This function delegates to `rav1e_sad_4x4_hbd_ssse3` and friends. The high bit depth (hbd) label follows dav1d convention. In libaom, highbd is more commonly used.

Reply:
:+1: done